src/App.tsx

```diff
@@ -4,6 +4,8 @@ import { theme } from "@pagopa/mui-italia";
 import React from "react";
 import { useTranslation } from "react-i18next";
 import { BrowserRouter, Navigate, Route, Routes } from "react-router-dom";
+import type { MuiLoadingButton } from '@mui/lab/themeAugmentation';
```

Comment:
To avoid the typescript error you can change this line to:
import "@mui/lab/themeAugmentation";